<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewExperimentWindow</class>
 <widget class="QWidget" name="NewExperimentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Particles in Box</string>
  </property>
  <widget class="QGroupBox" name="group_particle_settings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Particle settings</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>200</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="form_particle_settings">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_n_right">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Count (right side)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_particle_r">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_n_left">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Count (left side)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_v_init">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Initial speed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_v_loss">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Speed loss factor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="n_left"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="n_right"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="particle_r"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="v_init"/>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="v_loss"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_geometry">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Box geometry</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>200</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="form_box_geometry">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_box_width">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_box_height">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_barrier_x">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Barrier X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_barrier_width">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Barrier width</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_hole_y">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Hole Y</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_hole_height">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Hole height</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="box_width"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="box_height"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="barrier_x"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="barrier_width"/>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="hole_y"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="hole_height"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_collision_settings">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Collision speed change</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>200</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="form_collision_settings">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_delta_v_top">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Top</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_delta_v_bottom">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Bottom</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_delta_v_side">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Sides</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="delta_v_top"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="delta_v_bottom"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="delta_v_side"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_run">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
